app/views/devise/registrations/edit.html.erb

```diff
@@ -54,32 +53,32 @@
         </div>
         <div class="help-container">
           <p>
-            Confirm your password.
+            <%= t('.confirm_your_password')%>
           </p>
         </div>
       </div>
 
       <div class="form-row">
         <div class="field-container">
-          <%= f.label :current_password, class: 'control-label' %><br />
+          <%= f.label :current_password, class: 'control-label' %> <i>(<%= t('.we_need_your_current_password_to_confirm_your_changes', :default => 'we need your current password to confirm your changes') %>)</i><br />
           <%= f.password_field :current_password, class: 'text-field form-control' %>
         </div>
         <div class="help-container">
           <p>
-            Your current password is needed to confirm your changes.
+            <%= t('.current_password_required')%>
           </p>
         </div>
       </div>
-    </div>
-  </div>
 
-  <div class="row margin-left-none" style="margin-top: 20px">
-    <%= f.submit "Update", class: 'btn btn-primary col-md-2' %>
+      <div class="row margin-left-none" style="margin-top: 20px">
+        <%= f.submit t('.update', :default => "Update"), class: 'btn btn-primary col-md-2' %>
+      </div>
+    </div>
   </div>
 <% end %>
 
-<hr />
+<h2><%= t('.cancel_my_account', :default => 'Cancel my account') %></h2>
 
-<h2>Cancel my account</h2>
+<p><%= t('.unhappy', :default => 'Unhappy') %>? <%= link_to t('.cancel_my_account', :default => "Cancel my account"), registration_path(resource_name), :data => { :confirm => t('.are_you_sure', :default => "Are you sure?") }, :method => :delete %>.</p>
```

Comment:
This was auto-generated by devise-i18n-views, which I suppose knows better than I what's the best in this case. Probably why it has changed from link to button.

I agree to restore the ```danger``` style back.